update systemd docs (#799)
falcosecurity/falco-website: content/en/docs/getting-started/installation.md

The most secure way to run Falco is to install Falco directly on the host system so that Falco is isolated from Kubernetes in the case of compromise.
Then the Falco alerts can be consumed through read-only agents running in Kubernetes.

You can also run Falco directly in Kubernetes as a Daemonset using Helm, see the [third-party integrations](../third-party)
{{% /pageinfo %}}

There are 2 main ways to install Falco on your host:

1. Falco packages (`.deb`, `.rpm`)
2. Falco binary (`.tar.gz`)

## Falco packages

{{% pageinfo color="secondary" %}}

> The Falco packages shipped with `Falco 0.34` support for the first time other drivers besides the kernel module. The new Systemd units' names are:

Reviewer: `systemd` is the right name, without the uppercase `S`

Author: ty!

```bash
FALCO_FRONTEND=noninteractive apt-get install -y falco
```

Let's see an example of how to install the package in a Debian-like system, for example, `Ubuntu`.

1. Trust the `falcosecurity` GPG key

    ```bash
    curl -s https://falco.org/repo/falcosecurity-packages.asc | apt-key add -
    ```

2. Configure the apt repository

    ```bash
    echo "deb https://download.falco.org/packages/deb stable main" | tee -a /etc/apt/sources.list.d/falcosecurity.list
    ```

3. Update the package list

    ```bash
    apt-get update -y
    ```

4. Install some required dependencies that are needed to build the kernel module and the BPF probe

    ```bash
    apt install -y dkms make linux-headers-$(uname -r)
    # If you use the falco-driver-loader to build the BPF probe locally you need also clang toolchain
    apt install -y clang llvm
    # You can install also the dialog package if you want it
    apt install -y dialog
    ```

    > _Note_: You don't need to install these deps if you want to the modern BPF probe

5. Install the Falco package

    ```bash
    apt-get install -y falco
    ```

#### Installation with dialog

> If you have the `dialog` binary installed on your system, you should see something similar to this:

Reviewer: ```suggestion
If you have the `dialog` binary installed on your system, you will be prompted with this:
```

Reviewer: > you should see something similar to this:

Seems like there is some chance that you might see something different...

![](/docs/getting-started/images/systemd_dialog_1.png)

From here you can choose one of our 3 drivers `Kmod`, `eBPF`, `Modern eBPF` or a [`Manual configuration`](#installation-without-dialog-manual-configuration).

Here we select the `Kmod` case as an example. After the first dialog, you should see a second one:

![](/docs/getting-started/images/systemd_dialog_2.png)

> [Falcoctl](https://github.com/falcosecurity/falcoctl) is a tool revamped with `Falco 0.34` that offers shiny new features! One of the most important is the [automatic rulesets update]()!

Reviewer: Too many `!` in this paragraph imho

Author: agree with that!
